src/features/events_list/components/EventsPanel/EventsPanel.tsx

```diff
@@ -0,0 +1,87 @@
+import { Panel, PanelIcon } from '@konturio/ui-kit';
+import { useCallback, useState } from 'react';
+import clsx from 'clsx';
+import { Disasters24 } from '@konturio/default-icons';
+import { useAtom } from '@reatom/react';
+import { i18n } from '~core/localization';
+import { userResourceAtom } from '~core/auth';
+import { AppFeature } from '~core/auth/types';
+import { IS_MOBILE_QUERY, useMediaQuery } from '~utils/hooks/useMediaQuery';
+import { useAutoCollapsePanel } from '~utils/hooks/useAutoCollapsePanel';
+import { panelClasses } from '~components/Panel';
+import { useHeightResizer } from '~utils/hooks/useResizer';
+import { MIN_HEIGHT } from '../../constants';
+import { ShortState } from '../ShortState/ShortState';
+import { FullState } from '../FullState/FullState';
+import s from './EventsPanel.module.css';
+import type { ShortStateListenersType } from '@konturio/ui-kit/tslib/Panel';
+
+export function EventsPanel({
+  currentEventId,
+  onCurrentChange,
+}: {
+  currentEventId?: string | null;
+  onCurrentChange: (id: string) => void;
+}) {
+  const [panelState, setPanelState] = useState<'full' | 'short' | 'closed'>('full');
+  const isOpen = panelState !== 'closed';
+  const isMobile = useMediaQuery(IS_MOBILE_QUERY);
+  const [{ data: userModel }] = useAtom(userResourceAtom);
+  const handleRefChange = useHeightResizer(
+    (isOpen) => !isOpen && setPanelState('closed'),
+    isOpen,
+    MIN_HEIGHT,
+  );
+
+  const shortStateListeners: ShortStateListenersType = {
+    onClose: () => setPanelState('closed'),
+    onFullStateOpen: () => setPanelState('full'),
+    onShortStateOpen: () => setPanelState('short'),
+  };
+
+  const onPanelIconClick = useCallback(() => {
+    setPanelState('full');
+  }, [setPanelState]);
+
+  const onPanelClose = useCallback(() => {
+    setPanelState('closed');
+  }, [setPanelState]);
+
+  useAutoCollapsePanel(isOpen, onPanelClose);
+
+  return (
+    <div className={clsx(s.panelContainer, s[panelState])}>
```

Comment:
do you really need s[panelState] here? 
I can't see even 1 case you use it in EventsPanel.module.css

Reply:
hmm, smth weird with mine or yours .css file

Reply:
![image](https://user-images.githubusercontent.com/50058726/204758103-be8bafa7-72e1-4d65-84cc-030696647490.png)
![image](https://user-images.githubusercontent.com/50058726/204758219-1eb2129b-1012-4d48-8731-0e5932c73e79.png)